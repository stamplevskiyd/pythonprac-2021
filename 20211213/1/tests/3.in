[27, 8, 73, 47, 17, 65, 57, 100, 44, 99, 29, 80, 46, 62, 13, 96]
