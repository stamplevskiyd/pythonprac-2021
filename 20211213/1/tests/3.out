[8, 13, 17, 27, 29, 44, 46, 47, 57, 62, 65, 73, 80, 96, 99, 100]
